src/neural/xla/onnx2hlo.cc

```diff
@@ -288,6 +289,8 @@ pblczero::XlaShapeProto::Type OnnxTypeToXlaType(
       return pblczero::XlaShapeProto::C128;
     case pblczero::TensorProto::BFLOAT16:
       return pblczero::XlaShapeProto::BF16;
+    case pblczero::TensorProto::FLOAT8E5M2:
+      return pblczero::XlaShapeProto::F8E5M2;
     case pblczero::TensorProto::STRING:
```

Comment:
I believe I left this STRING line by mistake, feel free to remove it.
Also I didn't list fp8 types there because our onnx.proto didn't have ones. Now it has 4 of them, it may makes sense to add them all here (totally optional)